I-D-Profile-Negotiation.xml

```diff
@@ -282,94 +282,110 @@ Preference-Applied: profile=&lt;urn:example:schema&gt;
+			<t>The "Accept-Profile" and "Content-Profile" header field
+				do not contain the actual profile but instead points to it using a URI.
+				As long as the URI 	is only used to denote the profile,
+				it does not need to point to an	actual document but can be considered opaque.
+				If the parties involved agree on a profile definition,
+				the profile can be identified with e. g. a URN or an info-URI.
+				When a protocol-based URI is used, such as an FTP- or an HTTP-URI	is used, however,
+				it is RECOMMENDED that it dereference to a document
+				containing the profile definition. 
 			</t>
 			<section title="Accept-Profile Header Syntax">
 				<t>The "Accept-Profile" header field is used to specify one or more
 					content profiles the issuing agent can accept for processing.
-					Each profile is identified by a URI reference or -- e. g.
-					in the case of namespace-specific XML schemas (cf. 
-					<xref target="XMLSCHEMA">4.3.2 of XML Schema 1.1-1</xref>) -- a list of pairs of
-					URI references separated by whitespace similar to the syntax used in the xsi:schemaLocation attribute.
-					If several profiles are specified, quality values as defined in <xref target="RFC7230">Section 5.3.1 of
-					RFC 7230</xref> can be used to assign a relative "weight" to the preference.
+					Each profile is identified by a URI reference inside angle brackets ("&lt;&gt;").
+					URI references MUST be absolute URIs.
+					If several profiles are specified,
+					quality values as defined in <xref target="RFC7230">Section 5.3.1 of RFC 7230</xref>
+					can be used to assign a relative "weight" to the preference.
 					Exactly how that weight is used to determine the best representation
 					is beyond the scope of this specification.</t>
 				<t>A request without any "Accept-Profile" header field implies that the user agent
-					will accept content conforming to any profile. If the header field is
-					present in a request and the origin server cannot provide a representation
-					that conforms to the specified profile, it can either honour the header field by sending a 406 (Not
-					Acceptable) response or disregard the header field by treating the
-					response as if it is not subject to content negotiation. If the the origin server 
-					chooses to disregard the header field and the profile the content conforms to is known,
-					the origin server SHOULD send a "Profile" header indicating that profile
-					together with the payload.</t>
-				<t><xref target="abnf_accept_schema"/> describes the syntax (Augmented Backus-Naur Form) of the
-					header fields, using the grammar defined in <xref target="RFC5234">RFC 5234</xref> and the rules
+					will accept content conforming to any profile.
+					If the header field is present in a request
+					and the origin server cannot provide a representation
+					that conforms to the specified profile,
+					it can either honour the header field by sending a 406 (Not	Acceptable) response
```

Comment:
we'll need to agree on UK or US spelling

Reply:
I guess that RFCs default to US spelling...

Reply:
I've replaced all "honour" with "honor"